react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
         defaultValue={this.props.value}
-        ref={(inputElement) => (this.inputElement = inputElement)}
+        ref={inputElement => this.inputElement = inputElement} //eslint-disable-line
```

Comment:
The linter complains about assignment to an arrow function, so i disabled the line